.devcontainer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM docker.io/geosx/ubuntu20.04-gcc10:203-819
+
+RUN apt-get update
+RUN apt-get install -y \
+    ca-certificates \
+    curl \
+    git-lfs \
```

Comment:
If `git-lfs` is only used for integratedTests (and we do not take them here) should we add it to `Dockerfile` ?

Reply:
I think we do, because the idea is to provide a full dev env as fast as possible.
Having `git-lfs` installed can be convenient in that regard.
The idea is not to have the tiniest container here imo.